src/config.rs

```diff
@@ -27,6 +27,9 @@ pub struct StorageOptions {
 
     #[structopt(long, env = "RUSTUS_DIR_STRUCTURE", default_value = "")]
     pub dir_structure: String,
+
+    #[structopt(long, env = "RUSTUS_PREALLOCATE")]
```

Comment:
```suggestion
    #[structopt(long, env = "RUSTUS_PREALLOCATE", default_value = "true")]
```

Reply:
I thought about that, but it's a breaking change for cli users

Reply:
But you have to provide default. 
To turn it into minor change you can set default value to false.

Reply:
Am I missing something from the [docs](https://docs.rs/structopt/latest/structopt/#type-magic)? For me it looks like boolean variables are default to false, and then turn true if the flag is present